### PR TITLE
ci: add missing apps and modules

### PR DESCRIPTION
**Pull request by marie-j**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | yes
| New feature?     |no
| Breaking change? |no
| Tickets          | Fix #... 
| License          | BSD 3-Clause

### CODE REVIEW

**Review by JeremyDec — APPROVED** [review]

**Review by frenautvh — APPROVED** [review]

File: .github/labeler.yml [hunk 1]
```diff
@@ -21,13 +21,15 @@ universe-public-cloud:
...
   - packages/manager/apps/dedicated/**/!(dedicatedCloud)/**
+  - packages/manager/*/billing/**
   - packages/manager/*/cda/**
   - packages/manager/*/cloud-connect/**
   - packages/manager/*/dbaas-logs/**
   - packages/manager/*/enterprise-cloud-database/**
   - packages/manager/*/iplb/**
   - packages/manager/*/metrics/**
   - packages/manager/*/nasha/**
+  - packages/manager/*/netapp/**
```
Comment (antleblanc) [inline .github/labeler.yml:32]: Following location doesn't exist yet on `master` branch. Is it required?

**Review by antleblanc — APPROVED** [review]
Changes looks good to me.

**thought:** Commit type `ci` can be used instead. Isn't it?
see: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#type

File: .github/labeler.yml (hunk 1 above)
Comment (marie-j) [reply .github/labeler.yml:32]: If I'm correct configuration is only taken into account if done on `master` branch so any PR related to this topic won't get the labels if not added

**Comment by marie-j** [comment]
> Changes looks good to me.
> 
> **thought:** Commit type `ci` can be used instead. Isn't it?
> see: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#type

Sure will be udpated during "Squash and merge" :)

**Comment by antleblanc** [comment]
Update target branch to `release/2021-w31`.

**Pull request merged**